presto-hive/src/test/java/com/facebook/presto/hive/metastore/thrift/TestStaticHiveCluster.java

```diff
@@ -79,7 +92,14 @@ private static void assertCreateClientFails(HiveCluster cluster, String message)
 
     private static HiveCluster createHiveCluster(StaticMetastoreConfig config, List<HiveMetastoreClient> clients)
```

Comment:
I think this method should accept now `config` parameter

Reply:
do you mean like this ?
```private static HiveCluster createHiveCluster(StaticMetastoreConfig config, List<HiveMetastoreClient> clients, HiveClientConfig hiveClientConfig)```